go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/containerd/btrfs v1.0.0
 	github.com/containerd/cgroups v1.0.2
 	github.com/containerd/console v1.0.3
-	github.com/containerd/containerd/api v1.6.0-beta.1
+	github.com/containerd/containerd/api v1.6.0-beta.3
```

Comment:
The other go.mod files probably need to be updated as well, and "go mod tidy / go mod vendor" to be run to update the modules.txt

(like the previous update https://github.com/containerd/containerd/commit/37720fc6fdae4daf3c79794ad68f0e2c821f617c